sdk/monitor/query/azlogs/tsp-location.yaml

```diff
@@ -0,0 +1,3 @@
+directory: specificationmonitor/Monitor.Query.Logs
+commit: f1dc99dad0b0d61849393cfa14604c1a1a7b729b
+repo: Azure/azure-rest-api-specs
```

Comment:
```suggestion
repo: Azure/azure-rest-api-specs

```

Reply:
fixed in [c8d3d56](https://github.com/Azure/azure-sdk-for-go/pull/24955/commits/c8d3d56885e404e515bb9d2c9c9f2e32cf6ea03b)